tests/data_context/store/test_query_store.py

```diff
@@ -14,7 +14,7 @@ def basic_sqlalchemy_query_store(titanic_sqlite_db):
 
 
 @pytest.fixture()
-def test_basic_sqlalchemy_query_store_connection_string(
+def basic_sqlalchemy_query_store_connection_string(
```

Comment:
Unrelated change, just saw an issue in the same file. Fixtures shouldn't start with `test_`

Reply:
(just trying to keep the campsite cleaner than I found it 😄)